src/package/pywinrt/Microsoft.Windows.PyWinRT.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Windows.PyWinRT</id>
+    <version>1.0.0.0</version>
```

Comment:
0.9?

Reply:
offline conversation, this is overwritten by cmake file, so this is just a dummy value anyway.